CloudDeployment.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+
+call "%~dp0\Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat" ThirdPersonShooterServer Linux Development ThirdPersonShooter.uproject
```

Comment:
Why not build win64 here too?

Reply:
You mean, to build a client too? I'm thinking it will probably be best to separate the bat file calls from the actual deployment launch because we will have different ways to launch and it will probably be easier to write those in shell scripts anyway.

Reply:
I'll update this in the PR which adds metrics

Reply:
Roger